Remove SystemOrganisation global

jecisc opened the pull request:
We have a SystemOrganization global variable (for now... It's the next one to kill on my list) but a type in the Bootstrap creates a SystemOrganisation (with an s instead of the z) global.

This removes this unwanted global.

jecisc committed:
Remove SystemOrganisation global

We have a SystemOrganization global (for now... It's the next one to kill on my list) variable but a type in the Bootstrap creates a SystemOrganisation (with an s instead of the z) global.

This removes this unwanted global.

privat commented on src/Pharo30Bootstrap/PBImageBuilder50.class.st:
```diff
@@ -289,7 +289,7 @@ PBImageBuilder50 >> createInitialObjects [
 		'Smalltalk globals at: #Transcript put: nil.'.
 	self bootstrapInterpreter evaluateCode:
-		'Smalltalk globals at: #SystemOrganisation put: nil.'.
+		'Smalltalk globals at: #SystemOrganization put: nil.'.
```
I think you should just remove it since it worked without the correct name

jecisc replied:
Using the right global seems to break the bootstrap anyway so let's remove it yeah

Ducasse commented:
Build is broken

jecisc committed:
Remove useless global initialization

This global initialization seems useless and anyway, I'll kill SystemOrganization soon

MarcusDenker commented:
We might add a special DeprecatedGlobal later to warn clients that use it, but for early Pharo12 this is ok to remove it
